src/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -62,32 +63,32 @@ type Builder interface {
 func NewHostMonitoring(instance *dynatracev1beta1.DynaKube, clusterId string) Builder {
 	return &HostMonitoring{
 		builderInfo{
-			instance:       instance,
+			dynakube:       instance,
 			hostInjectSpec: instance.Spec.OneAgent.HostMonitoring,
-			clusterId:      clusterId,
-			deploymentType: DeploymentTypeHostMonitoring,
+			clusterID:      clusterId,
+			deploymentType: deploymentmetadata.DeploymentTypeHostMonitoring,
```

Comment:
these constants are kind of redundant in combination with the package name, maybe `deploymentmetadata.HostMonitoringKey`

Reply:
but its not a key :)
its actual the 2. part of the `OrchestrationTech`
the word deployment is the same but you cant remove it without causing more confusion IMO
- `deploymentmetadata.TypeHostMonitoring` kinda refers to the type of deploymentmetadata and not type of deployment
- `deploymentmetadata.HostMonitoringKey` key for what?